hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/ExampleDriver.java

```diff
@@ -1,15 +1,26 @@
 /**
- * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why have you deleted the Dell license header from this file? I thought that the idea was to change it only on those files got from Hadoop repository that are mostly identical, right?

Reply:
Added back - now it carries both licenses, one for dell, another for the apache hadoop.